text/0032-nft-standard.md

```diff
@@ -93,13 +122,66 @@ The AttributeObject has the following JSON structure:
 
 The AttributeObject may have more attributes than the ones described above but those are the required ones.
 
+- The RoyaltyObject has the following JSON structure:
+
+```
+"creator_royalty_basis_points": {
+    "type": "integer",
+    "description": "Royalties percentage awarded to creators (basis points), e.g. 500"
```

Comment:
I think would be good to add the % here as well. "..., e.g. 500, which means 5%"